styles.css

```diff
@@ -0,0 +1,163 @@
+body {
+	font-family: 'Old Standard TT', serif;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/burger-one/pullRequest?prid=6050239)